hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeRecordReader.java

```diff
@@ -104,11 +104,12 @@
   private Configuration hadoopConf;
 
   @BeforeEach
-  public void setUp() {
+  public void setUp() throws IOException {
     hadoopConf = HoodieTestUtils.getDefaultHadoopConf();
     baseJobConf = new JobConf(hadoopConf);
     baseJobConf.set(HoodieRealtimeConfig.MAX_DFS_STREAM_BUFFER_SIZE_PROP, String.valueOf(1024 * 1024));
-    fs = FSUtils.getFs(basePath.toString(), baseJobConf);
```

Comment:
you can simply do `basePath.toUri()` ? this was set by `@TempDir` so it's always `/tmp/xxx`. we don't need to check scheme